cosmic_ray/timing.py

```diff
@@ -18,6 +18,7 @@ class Timer:
     """
 
     def __init__(self):
+        self._start = None
```

Comment:
This seems unnecessary.

Reply:
it is according to Python, but not according pylint, otherwise the linter complies that we assign to something that was not first defined in init